### PR TITLE
Aquahamzah Pull Request (Changed image!)

### PR DESCRIPTION
**Pull request by aquahamzah**
# Application to the Class of 2020🎓

This pull request template helps you complete an application to the **Class of 2020**. Use the checklist below to verify you have followed the instructions correctly. 

## Checklist ✅

- [x] I have read the instructions on the README file before submitting my application. 
- [x] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`), added my profile picture and markdown file.
- [x] I have used the Markdown file template to add my information to the Year Book.
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the photo added to the template will be used in the ceremony "Graduate Walk". 
- [x] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.

### CODE REVIEW

**Comment by aquahamzah** [comment]
Was just wondering if the goodies are still up for grabs as this is my second pull request and my first one I ticked of all three stages (making dev post, wanting it to shown in the ceremony, e.t.c) - but was told to make another as the image PX size was a little small!

File: _data/Aquahamzah/Aquahamzah.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: Aquahamzah
+institution: City, University of London
+profile_pic: aquahamzah.jpg
+quote: His StackOverflow requests...it's over 9000!
+github_user: aquahamzah
+---
+
```
Comment (Mohammed-Shoaib) [inline _data/Aquahamzah/Aquahamzah.md:8]: Can you please remove the extra new line?

File: _data/Aquahamzah/Aquahamzah.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+name: Aquahamzah
+institution: City, University of London
+profile_pic: aquahamzah.jpg
```
Comment (Mohammed-Shoaib) [inline _data/Aquahamzah/Aquahamzah.md:4]: Incorrect file name, please do one of the following:
1. Change `aquahamzah.jpg` to `aquahamzah.png` in the markdown file.
2. Rename `aquahamzah.png` to `aquahamzah.jpg`.
I'd suggest you make the change in the markdown file!

File: _data/Aquahamzah/Aquahamzah.md (hunk 1 above)
Comment (aquahamzah) [reply _data/Aquahamzah/Aquahamzah.md]: Hi there, just wondering if that shows up correctly for you now? Not entirely sure why it had the extra line! Apologies.

File: _data/Aquahamzah/Aquahamzah.md (hunk 2 above)
Comment (aquahamzah) [reply _data/Aquahamzah/Aquahamzah.md]: Done this! My apologies again, forgot to change the .md file to match the new file upload for my profile picture. :)

**Pull request merged**